index.html

```diff
@@ -210,8 +210,17 @@ <h4 class="col s12 light header center">Checkout the MoSDeF Ecosystem</h4>
                         </div>
                     </div>
                 </div>
+                <div class="col s12 l4 offset-l4">
+                    <div class="card small banner-background">
+                        <div class="card-content white-text">
+                            <div class="card-title">MoSDeF Workflows</div>
+                            <p>Curated demo workflows using MoSDeF the framework</p>
```

Comment:
```suggestion
                            <p>Curated demo workflows using the MoSDeF framework</p>
```
minor rewording.
It might also be nice if we mention that these branch out into various simulation engines.